analytics-datastore-elastic-search/swarm.sh

```diff
@@ -1,172 +1,221 @@
-  try "docker config create --label name=elasticsearch ${TIMESTAMP}-es01.crt ./certs/es01/es01.crt" "Error creating config es01.crt"
-  try "docker config create --label name=elasticsearch ${TIMESTAMP}-es01.key ./certs/es01/es01.key" "Error creating config es01.key"
-  try "docker config create --label name=elasticsearch ${TIMESTAMP}-es02.crt ./certs/es02/es02.crt" "Error creating config es02.crt"
-  try "docker config create --label name=elasticsearch ${TIMESTAMP}-es02.key ./certs/es02/es02.key" "Error creating config es02.key"
-  try "docker config create --label name=elasticsearch ${TIMESTAMP}-es03.crt ./certs/es03/es03.crt" "Error creating config es03.crt"
-  try "docker config create --label name=elasticsearch ${TIMESTAMP}-es03.key ./certs/es03/es03.key" "Error creating config es03.key"
-
-  log info "Updating es-01 with certs"
-  try "docker service update \
-    --config-add source=${TIMESTAMP}-ca.crt,target=/usr/share/elasticsearch/config/certs/ca/ca.crt \
-    --config-add source=${TIMESTAMP}-es01.crt,target=/usr/share/elasticsearch/config/certs/es01/es01.crt \
-    --config-add source=${TIMESTAMP}-es01.key,target=/usr/share/elasticsearch/config/certs/es01/es01.key \
-    --replicas 1 \
-    instant_analytics-datastore-elastic-search-01" "Error updating es01"
-
-  log info "Updating es-02 with certs"
-  try "docker service update \
-    --config-add source=${TIMESTAMP}-ca.crt,target=/usr/share/elasticsearch/config/certs/ca/ca.crt \
-    --config-add source=${TIMESTAMP}-es02.crt,target=/usr/share/elasticsearch/config/certs/es02/es02.crt \
-    --config-add source=${TIMESTAMP}-es02.key,target=/usr/share/elasticsearch/config/certs/es02/es02.key \
-    --replicas 1 \
-    instant_analytics-datastore-elastic-search-02" "Error updating es02"
-
-  log info "Updating es-03 with certs"
-  try "docker service update \
-    --config-add source=${TIMESTAMP}-ca.crt,target=/usr/share/elasticsearch/config/certs/ca/ca.crt \
-    --config-add source=${TIMESTAMP}-es03.crt,target=/usr/share/elasticsearch/config/certs/es03/es03.crt \
-    --config-add source=${TIMESTAMP}-es03.key,target=/usr/share/elasticsearch/config/certs/es03/es03.key \
-    --replicas 1 \
-    instant_analytics-datastore-elastic-search-03" "Error updating es03"
+function add_docker_configs() {
+  local -r TIMESTAMP="$(date "+%Y%m%d%H%M%S")"
+  local -r path_config_certs="/usr/share/elasticsearch/config/certs/"
+  log info "Creating configs..."
+
+  try "docker config create --label name=elasticsearch ${TIMESTAMP}-ca.crt ./certs/ca/ca.crt" catch "Error creating config ca.crt"
+
+  number_configs=(
+    "01"
+    "02"
+    "03"
+  )
+  for n in "${number_configs[@]}"; do
+    try "docker config create --label name=elasticsearch ${TIMESTAMP}-es$n.crt ./certs/es$n/es$n.crt" catch "Error creating config es$n.crt"
+    try "docker config create --label name=elasticsearch ${TIMESTAMP}-es$n.key ./certs/es$n/es$n.key" catch "Error creating config es$n.key"
```

Comment:
Should we not be throwing the error here

Reply:
done